package.json

```diff
@@ -4,12 +4,15 @@
   "description": "React / Webpack / Babel framework for quick set up",
   "main": "index.js",
   "scripts": {
-    "build": "webpack --config ./webpack.prod.config.js --progress --colors",
+    "clean": "rm -rf ./dist/*",
+    "build": "webpack --config ./webpack.config.js --progress --colors",
+    "build:prod": "NODE_ENV='production' webpack --config ./webpack.config.js --progress --colors",
     "start": "node ./app.js",
     "dev": "webpack-dev-server --inline --hot",
```

Comment:
@cisacke good point. I've kept the dev server like this because it provides much better error messages than the styleguide build/server. Pretty dumb reason to keep it, but until I find a better way to log errors, I'm going to hold on to this